pkg/controllers/subnet/subnet_controller.go

```diff
@@ -83,11 +83,16 @@ func (r *SubnetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 				updateFail(r, &ctx, obj, "")
 				return ResultRequeue, err
 			}
+
 			if obj.Spec.AccessMode == "" {
-				obj.Spec.AccessMode = v1alpha1.AccessMode(vpcNetworkConfig.DefaultSubnetAccessMode)
+				accessMode := v1alpha1.AccessMode(v1alpha1.AccessModePrivate)
```

Comment:
For subnet, we don't need to set PodSubnetAccessMode, since there is no pod subnet created.
subnet  is only VM.

For POD, we only have pod-default subnetset. You can confirm with @lxiaopei

Reply:
according to NSX API doc, if no access mode, then Private access mode will be used.